pywr/_core.pyx

```diff
@@ -57,9 +57,11 @@ cdef class ScenarioCollection:
         def __get__(self):
             if len(self._scenarios) == 0:
                 return (1, )
-            return (sc._size for sc in self._scenarios)
+            return tuple(sc.size for sc in self._scenarios)
 
-    cpdef int ravel_indices(self, int[:] scenario_indices):
+    cpdef int ravel_indices(self, int[:] scenario_indices) except? -1:
```

Comment:
Is `-1` a valid return from this function if an exception hasn't occured? If not the `?` isn't required.